tests/unit/resources/test_policies.py

```diff
@@ -141,6 +141,20 @@ def test_get_vms_not_found(self, mock_get):
 
         mock_get.assert_called_once_with('/policies/ABCDE/virtual_machines')
 
+    @mock.patch.object(Connection, "post")
+    @mock.patch.object(Connection, "get")
+    def test_create_policy(self, mock_post, mock_get):
```

Comment:
def test_create_policy(self, mock_get, mock_post)

Reply:
Updated the code as per review comment